sql_queries.py

```diff
@@ -56,6 +62,7 @@
                          "discord_id TEXT NOT NULL, " \
                          "semester TEXT NOT NULL, " \
                          "lva_nr TEXT NOT NULL, " \
+                         "active INTEGER NOT NULL, " \
```

Comment:
@xMuffin now I get your intention: [Sqlite3 Booleans](https://www.sqlite.org/datatype3.html#boolean_datatype)
Instead of INTEGER use [BOOLEAN](https://www.sqlite.org/datatype3.html#affinity_name_examples), still only a NUMERIC, but it should recognize TRUE and FALSE. Play around with it!